### DISCUSSION TITLE
how can I get set the credentials?

### DISCUSSION THREAD

**Discussion by Romerolweb** [saleweaver/python-amazon-sp-api#478] [Q&A] [answered]
I am trying to run a script, And have not been able to set up the credentials once I import the package. Do I need to modify the source, or can I do it in my script?

**Comment by saleweaver** [comment, marked as answer]
Hi, here's extensive documentation: https://sp-api-docs.saleweaver.com/credentials/

**Reply by scoroman** [reply]
Link no longer works. Updated link? :)

**Reply by jplusc** [reply]
https://python-amazon-sp-api.readthedocs.io/en/latest/credentials.html
